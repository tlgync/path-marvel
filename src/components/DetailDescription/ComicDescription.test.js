import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { DetailDescription } from '.';

test('DetailDescription Render SnapShot', () => {
  const description = '-';
  const tree = renderer.create((
    <Router>
      <DetailDescription description={description} />
    </Router>
  )).toJSON();

  expect(tree).toMatchSnapshot();
});
