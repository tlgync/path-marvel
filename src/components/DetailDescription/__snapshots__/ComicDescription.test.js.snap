// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailDescription Render SnapShot 1`] = `
<div
  className="detail-desc"
>
  <span>
    Description
  </span>
  <span>
    -
  </span>
</div>
`;
